### DISCUSSION TITLE
Integrating PK-Sim Parameters into an R-Based Program

### DISCUSSION THREAD

**Discussion by ZetaWilliam** [Open-Systems-Pharmacology/Forum#1734] [Questions & Problems]
Dear PK-Sim Community,

I am currently exploring the capabilities of PK-Sim and am interested in automating the retrieval of physiological parameters for individuals of different species, taking into account variables such as age and weight. Specifically, I am seeking a method to establish a program based on R that could directly interface with the PK-Sim database to extract this information.

Could anyone guide me on how to achieve this integration? Are there any existing R packages or APIs that facilitate communication with the PK-Sim database for such purposes?

Any pointers or examples to similar implementations would be immensely appreciated. I am looking forward to your suggestions and assistance.

Thank you for your support.

**Comment by Yuri05** [comment]
Please check the [documentation of the ospsuite R package](https://www.open-systems-pharmacology.org/OSPSuite-R/), particularly [Creating individuals](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/create-individual.html) and [Creating populations](https://www.open-systems-pharmacology.org/OSPSuite-R/articles/create-run-population.html)
